pwiz_tools/Skyline/Controls/Startup/StartPage.cs

```diff
@@ -637,6 +638,23 @@ private void Tutorial(string skyFileLocation, string pdfFileLocation, string zip
         {
             Assume.IsNotNull(skyFileLocation);
 
+            // Test support
+            if (TutorialActionsTestDataUrisOnly)

```

Comment:
I'd rather see the test support limited to an accessor to an IEnumerable<Uri> and let the test itself do the HTTP access with an Assert.IsTrue(). I think the fact that you are using Assume.IsTrue() as a substitute for Assert.IsTrue() is a dead give-away that you are doing too much in the main project. Assume is not supposed to be for testing.